go.mod

```diff
@@ -4,6 +4,7 @@ go 1.15
 
 require (
 	github.com/AlecAivazis/survey/v2 v2.2.8
+	github.com/andygrunwald/go-jira v1.13.0 // indirect
```

Comment:
Can you try running `go mod tidy` to see if this cleans up the `// indirect`?

Thanks!

Reply:
Fixed.